src/mcp_server_helm/commands/release.py

```
@@ -0,0 +1,320 @@
+        notes = release_info.get('info', {}).get('notes')
+        if notes:
+            formatted_output += "\nNOTES:\n"
+            formatted_output += notes
+
+        return formatted_output
+    except json.JSONDecodeError:
+        # If output is not JSON, return raw output
+        return f"Installation output:\n{output}"
+
+
+def helm_upgrade(release_name: str, chart: str, namespace: Optional[str] = None,
```

Pylint (3.3.7)

[refactor] Too many arguments (10/5)

(R0913)